src/components/daySelector/_tests_/daySelector.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
there shouldn't be a `_tests` folder.
this file should be together with the file it's testing in the same folder.
the name of the folder should be the same name as the component so if it's upper case, then folder name as well